### PR TITLE
Version 9.4.0-rc1

### PR DESCRIPTION
**Pull request by beenje**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

Update recipe to test 9.4.0-rc1.
Added macOS support :-)

Note that this is to update the `rc` branch. Will not impact the default version.

### CODE REVIEW

**Comment by beenje** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,32 +48,26 @@ test:
...
-      - include/tango                       # [linux]
-      - Library/bin/tango.dll               # [win]
-      - Library/lib/tango.lib               # [win]
-      - Library/include/tango               # [win]
-      - Library/include/pkgconfig/tango.pc  # [win]
+outputs:                                                                 # [linux]
+  - name: cpptango                                                       # [linux]
+    files:                                                               # [linux]
+      - lib/libtango.so                                                  # [linux]
+      - lib/libtango.so.9                                                # [linux]
```
Comment (bourtemb) [inline recipe/meta.yaml:57]: If I'm not mistaken, I think it should be the following instead:
```suggestion
      - lib/libtango.so.94                                                # [linux]
```

Comment (beenje) [reply recipe/meta.yaml:57]: Thanks for catching this!

File: recipe/bld.bat [hunk 2]
```diff
@@ -7,12 +7,13 @@ cmake -G "NMake Makefiles" ^
...
-      -DOMNI_BASE:PATH="%LIBRARY_PREFIX%" ^
-      -DZMQ_BASE:PATH="%LIBRARY_PREFIX%" ^
+      -DTANGO_CPPZMQ_BASE:PATH="%LIBRARY_PREFIX%" ^
+      -DTANGO_IDL_BASE:PATH="%LIBRARY_PREFIX%" ^
+      -DTANGO_OMNI_BASE:PATH="%LIBRARY_PREFIX%" ^
+      -DTANGO_ZMQ_BASE:PATH="%LIBRARY_PREFIX%" ^
+      -DTANGO_JPEG_BASE:PATH="%LIBRARY_PREFIX%" ^
       -DPTHREAD_WIN:PATH="%LIBRARY_PREFIX%" ^
-      -DTANGO_JPEG_MMX=OFF ^
+      -DTANGO_USE_PCH=OFF ^
```
Comment (bourtemb) [inline recipe/bld.bat:16]: If you want to speed up the compilation, you can use `-DTANGO_USE_PCH=ON`

Comment (bourtemb) [reply recipe/bld.bat:16]: Well, I'm actually not sure it works on Windows ...

File: recipe/build.sh [hunk 3]
```diff
@@ -1,14 +1,13 @@
 mkdir build
 cd build
-cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+cmake ${CMAKE_ARGS} \
+      -DCMAKE_BUILD_TYPE=RelWithDebInfo \
       -DCMAKE_VERBOSE_MAKEFILE=ON \
-      -DCMAKE_INSTALL_PREFIX=$PREFIX \
-      -DCMAKE_INSTALL_LIBDIR=lib \
```
Comment (bourtemb) [inline recipe/build.sh:6]: Are these 2 lines no longer required?

On the [conda-forge  knowledge base](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=compiler%20cxx), it is written :

> Some optional, but useful CMake options:
        ...
        -DCMAKE_INSTALL_PREFIX=$PREFIX Specify the install location.
        -DCMAKE_INSTALL_LIBDIR=lib Libraries will land in $PREFIX/lib, sometimes projects install into lib64 or similar but on conda-forge we keep shared libraries in simply lib.

Comment (beenje) [reply recipe/build.sh:6]: Those 2 options are part of the `CMAKE_ARGS` variable. This is why I removed them.

**Pull request merged**